lib/constants.js

```diff
@@ -73,6 +73,8 @@ module.exports = {
     MONGO_ALARM: 'MONGO-ALARM',
     ORION_ALARM: 'ORION-ALARM',
     IOTAM_ALARM: 'IOTAM-ALARM',
+    ATTRIBUTE_DEFAULT,
+    DATETIME_DEFAULT,
```

Comment:
Should these constants have an actual value (as the others above)?

Reply:
Maybe it's done in L29 and L30 ?

```
const DATETIME_DEFAULT = '1970-01-01T00:00:00.000Z';
const ATTRIBUTE_DEFAULT = ' ';
```

Reply:
Correct - standard scope inheritance.

Reply:
NTC